docs/installation.rst

```diff
@@ -54,9 +54,9 @@ Docker setup
 
         cd kuma
 
-#. Initialize and customize ``.env``. Linux users should set the ``UID``
-   parameter in ``.env``, to avoid issues when mixing ``docker-compose`` and
-   ``docker`` commands::
+#. Initialize and customize ``.env``. Linux users should set the ``UID`` parameter in ``.env``
+   (i.e remove the "#" before ``UDI``,example: change ``#UDI=1000`` to ``UDI=1000``) to avoid 
+   issues when mixing ``docker-compose`` and ``docker`` commands::
```

Comment:
You may want to mention file permission issues as well.

Reply:
I think those issues are because of the UID so i dont think there is a need to mention it and we can mention about the Troubleshooting page here in case there is a problem during instillation. We can also mention about the other troubles, not using sudo and other things in the Troubleshooting  page.